packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -78,6 +88,16 @@ $card--pageContentWrapper-width           : 472px; // so that it falls on our 8p
 }
     .c-card-innerSpacing {
         padding: $card-padding;
+
+        &.c-card-innerSpacing--large {
+            padding: $card-padding-large $card-padding;
+        }
+
+        @include media('>=mid') {
+            &.c-card-innerSpacing--large {
+                padding: $card-padding-large;
+            }
+        }
```

Comment:
Late to this one, but just a small thing that you could write this like:

```scss
        &.c-card-innerSpacing--large {
            padding: $card-padding-large $card-padding;
            
            @include media('>=mid') {
                 padding: $card-padding-large;
            }
        }
```

Reply:
fixed https://github.com/justeat/fozzie-components/pull/1524